arangod/MMFiles/MMFilesCollection.cpp

```diff
@@ -2761,7 +2761,7 @@ void MMFilesCollection::truncate(transaction::Methods* trx,
 
       LocalDocumentId const documentId = LocalDocumentId::create();
       TRI_voc_rid_t revisionId;
-      newObjectForRemove(trx, oldDoc, documentId, *builder.get(), options.isRestore, revisionId);
```

Comment:
A new revisionID is generated to be put into the tombstone in MMFiles.
Id don't know why the "documentId" was here, I guess @jsteemann added it just in case